cmd/corectl/main.go

```diff
@@ -88,8 +91,18 @@ func configGenerator(db *sql.DB, args []string) {
 	flags.Parse(args)
 	args = flags.Args()
 
+	//not a blocksigner
```

Comment:
Sorry to be so nitpicky but I'd like to see a space after the `//` in all comments.